src/constructs/lambda/event-sources.ts

```diff
@@ -0,0 +1,93 @@
+import type { IEventSourceDlq } from "@aws-cdk/aws-lambda";
```

Comment:
This file is confusing me a little, can we discuss IRL?

Reply:
I think I've made all of the changes that we discussed now @akash1810 - let me know if I missed anything!